tests/test_component_as_view.py

```diff
@@ -150,14 +162,6 @@ def test_replace_slot_in_view(self):
             response.content,
         )
 
-    def test_replace_slot_in_view_with_insecure_content(self):
```

Comment:
Shouldn't we still test this?

Reply:
Ah, forgot to attach a comment here, but this is a duplicate, it has the same function body as the function below (`test_replace_slot_in_view_with_insecure_content`)

Reply:
But thanks for pointing out, as I'm looking at it now, I guess the first functions probavbly should have used `/test_context_insecure/` endpoint instead?

Reply:
Yes, you're right. I think I made a typo when I wrote that 😅

Reply:
Haha, happens :D Reverted the deletion and fixed the test in https://github.com/EmilStenstrom/django-components/pull/452/commits/8cc12ef4488500b87dbc0f411527bcc0872e35b8 :tada: